docs/UserGuide.md

```diff
@@ -221,12 +223,15 @@ editclass 1 [cn/CLASS_NAME] [ct/CLASS_TIMING] [r/RATE] [l/LOCATION]
 ```
 
 * Edits the class at the specified `INDEX`. The index refers to the index number shown in the displayed class
-  list in the `classes` tab. The index **must be a positive integer** 1, 2, 3, …​
+  list in the `classes` tab. 
+  * The index **must be a positive integer** 1, 2, 3, …​
+  * The index must also not be out of rage.
```

Comment:
spelling error range.

Reply:
maybe can elaborate what the range means also

Reply:
Ok i add in last part of notes about command format